UnitTests/BTPaymentMethodNonce+DropInTests.swift

```diff
@@ -3,61 +3,40 @@ import BraintreeDropIn
 
 class BTPaymentMethodNonceDropInTests: XCTestCase {
 
-    let sharedParser = BTPaymentMethodNonceParser.shared()
```

Comment:
`BTPaymentMethodNonceParser` is now internal to the base SDK, so we can't use it here.